app/views/mailers/mailshots_mailer/company_support_donor.html.erb

```diff
@@ -0,0 +1,606 @@
+          </div>
+        <!--[if (mso)|(IE)]></td></tr></table><![endif]-->
+        </div>
+      </div>
+      <div>
+        <div class="layout one-col fixed-width stack" style="margin: 0 auto;max-width: 600px;min-width: 320px; width: 320px;width: calc(28000% - 167400px);overflow-wrap: break-word;word-wrap: break-word;word-break: break-word;">
+          <div class="layout__inner" style="border-collapse: collapse;display: table;width: 100%;background-color: #ffffff;">
+          <!--[if (mso)|(IE)]><table align="center" cellpadding="0" cellspacing="0" role="presentation"><tr class="layout-fixed-width" style="background-color: #ffffff;"><td style="width: 600px" class="w560"><![endif]-->
+            <div class="column" style="text-align: left;color: #130B43;font-size: 16px;line-height: 25px;font-family: Poppins,Arial,'Helvetica Neue',Helvetica,sans-serif;">
+
+              <div style="margin-left: 20px;margin-right: 20px;">
+                <div style="mso-line-height-rule: exactly;mso-text-raise: 11px;vertical-align: middle;">
+                  <p style="margin-top: 16px;margin-bottom: 0;">
+                    Hi <%= @user.handle %>,
+                  </p>
+                  <p style="margin-top: 20px;margin-bottom: 0;">
+                    Thank you so much for donating to Exercism in the last 18 months. Thanks to support from people like you, we've been able to provide over 100 million minutes of free coding education and continue to maintain and grow the platform.
+                  </p>
+                  <p style="margin-top: 20px;margin-bottom: 0;">
+                    Now we need your help in keeping the lights on at Exercism for another year! <strong style="font-weight:600">But we're not asking you for more money.</strong> Instead, we're hoping you might be able to get your company to support us. A $1k donation is often a small amount for a company but a huge amount for us. Could you introduce us to someone at your company who could support Exercism?                    
+                  </p>
+                  <p style="margin-top: 20px;margin-bottom: 0;">
+                    Some people are also able to support Exercism through their corporate learning or education allowance. They're allowed to spend up to a certain amount on their professional development and choose to donate some of it to Exercism. Could you do the same?
+                  </p>
+                  <p style="margin-top: 20px;margin-bottom: 0;">
+                    As a reminder, Exercism is a not-for-profit organisation. We rely entirely on donations from individuals (like you!) and companies to survive. You can learn more about our impact on our <a href="https://exercism.org/about/impact" style="font-weight:600; color:#2E57E8; text-decoration: underline">impact page</a>.
```

Comment:
I've made one small tweak here.
In the source mail template, it said:

> You can learn more about our impact at https://exercism.org/about/impact 

However, this is an HTML email so we should probably be hyperlinking. I've changed it to:

> You can learn more about our impact on our <a href="https://exercism.org/about/impact" style="font-weight:600; color:#2E57E8; text-decoration: underline">impact page</a>.